src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -78,14 +78,48 @@ public sealed class HousingRegression
         /// <summary>
         /// Downloads the wikipedia detox dataset from the ML.NET repo.
         /// </summary>
-        public static string DownloadSentimentDataset()
-         => Download("https://raw.githubusercontent.com/dotnet/machinelearning/76cb2cdf5cc8b6c88ca44b8969153836e589df04/test/data/wikipedia-detox-250-line-data.tsv", "sentiment.tsv");
+        public static (string trainFile, string testFile) DownloadSentimentDataset()
```

Comment:
>(string trainFile, string testFile) [](start = 22, length = 35)

[https://github.com/dotnet/machinelearning/issues/2501](https://github.com/dotnet/machinelearning/issues/2501)

Can we not use value tuples? They nice, but are they really necessary?
Imagine we will need to write samples to F#, and we want to reuse this class. Would be quite bad.